Firebase/Core/FIRApp.m

```diff
@@ -809,26 +830,33 @@ + (nullable NSNumber *)readDataCollectionSwitchFromPlist {
 
 #pragma mark - Sending Logs
 
-- (void)sendLogsWithServiceName:(NSString *)serviceName
-                        version:(NSString *)version
-                          error:(NSError *)error {
+- (void)sendLogsWithServiceName:(NSString *)serviceName version:(NSString *)version {
```

Comment:
It'd be nice if we could take this out (or replace the body of sendLogsWithServiceName with a NOP/TODO/REMOVE)

Reply:
Removed.